exasol/toolbox/nox/_documentation.py

```diff
@@ -88,6 +105,117 @@ def clean_docs(_session: Session) -> None:
         shutil.rmtree(docs_folder)
 
 
+@nox.session(name="docs:links", python=False)
+def docs_list_links(session: Session) -> None:
+    """List all the links within the documentation."""
+    ignore = [r".*"]
+    env = os.environ.copy()
+    env["SPHINX_EXTRA_LINKCHECK_IGNORES"] = ",".join(ignore)
+    with tempfile.TemporaryDirectory() as path:
+        tmpdir = Path(path)
+        sp = subprocess.run(
+            [
+                "poetry",
+                "run",
+                "--",
+                "sphinx-build",
+                "-b",
+                "linkcheck",
+                PROJECT_CONFIG.root / "doc",
+                tmpdir,
+            ],
+            capture_output=True,
+            text=True,
+            env=env,
+        )
+        print(sp.returncode)
+        if sp.returncode >= 2:
```

Comment:
Could we do `sp.check_returncode()` instead? I'd also suggest creating a function to encase the overlap between `docs:links` and `docs:links:checks`. So namely the context manager with the command.